paper/paper.md

```diff
@@ -63,7 +63,7 @@ We envision many further applications within and outside of astrophysics.
 The functionality of `JAXbind` extends the external callback functionality in JAX.
 Currently, `JAXbind`, akin to the external callback functions in JAX, briefly requires python's global interpreter lock (GIL) to call the user specified python function.
 In contrast to JAX's external callback functions, it allows for both a custom Jacobian-vector product and vector-Jacobian product.
-To the best of our knowledge no other code currently exists for connecting generic functions and both of their Jacobian products to JAX.
+To the best of our knowledge no other code currently exists for easily binding generic functions and both of their Jacobian products to JAX, without the need for C++ or LLVM.
```

Comment:
I would drop the "easily" part here as this is already heavily implied by saying "without the need for C++ or LLVM". This is also within the #23 .